### ISSUE TITLE
error with squirrel_year1.launch (localizer?)

### ISSUE THREAD

**Issue by edith-langer** [squirrel-project/squirrel_nav#22]
Michael and I get the same error when trying to launch squirrel_year1.launch.

[ERROR] [1426688844.084783175, 1132.418000000]: /localizer: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[ WARN] [1426688843.751126017, 1132.350000000]: Waiting on transform from base_link to map to become available before running costmap, tf error: Could not find a connection between 'map' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.

@ipa-nhg @ipa-rmb @federico-b Any ideas?
